cmd/guacone/cmd/files.go

```diff
@@ -44,8 +44,11 @@ import (
 	"github.com/guacsec/guac/pkg/logging"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"golang.org/x/sync/errgroup"
 )
 
+const maxConcurrentJobsString string = "MAX_CONCURRENT_JOBS"
```

Comment:
can we add this to the help string as well?

Reply:
added!